docs/reference/release-notes/6.8.asciidoc

```diff
@@ -18,10 +22,42 @@ Machine Learning::
 [float]
 === Bug fixes
 
+Allocation::
+* Handle negative free disk space in deciders {pull}48392[#48392] (issue: {issue}48380[#48380])
+
+Analysis::
+* Actually close IndexAnalyzers contents {pull}43914[#43914]
+
+Authentication::
+* Remove unnecessary details logged for OIDC {pull}48271[#48271]
```

Comment:
This PR was closed, but not merged:

https://github.com/elastic/elasticsearch/pull/48271

We should remove it from the RNs.